index.html

```diff
@@ -1,299 +1,312 @@
+            </div>
+          </li>
+
+          <li class="card-details reverse">
+            <div class="image">
+              <img src="./photos/Portfolio2.png" alt="tonic" />
+            </div>
+            <div class="text-div">
+              <h2 class="title">Multi-Post Stories</h2>
+              <ul class="tech-btns">
+                <li>CANOPY</li>
+                <li class="common">Back End Dev</li>
+                <li class="common">2015</li>
+              </ul>
+
+              <p class="paragraph">
+                A daily selection of privately personalized reads no accounts or
+                sign-ups required.
+              </p>
+              <ul class="languages">
+                <li>Html</li>
+                <li>Css</li>
+                <li>Javascript</li>
+              </ul>
+              <a class="see-project" href="#">See Project</a>
+            </div>
+          </li>
+
+          <li class="card-details">
+            <div class="image">
+              <img src="./photos/Portfolio3.png" alt="tonic" />
+            </div>
+            <div class="text-div">
+              <h2 class="title">Tonic</h2>
+              <ul class="tech-btns">
+                <li>CANOPY</li>
+                <li class="common">Back End Dev</li>
+                <li class="common">2015</li>
+              </ul>
+
+              <p class="paragraph">
+                A daily selection of privately personalized reads no accounts or
+                sign-ups required.
+              </p>
+              <ul class="languages">
+                <li>Html</li>
+                <li>Css</li>
+                <li>Javascript</li>
+              </ul>
+              <a class="see-project" data-index-number="2" href=""
+                >See Project</a
+              >
+            </div>
+          </li>
+
+          <li class="card-details reverse">
+            <div class="image">
+              <img src="./photos/Portfolio2.png" alt="tonic" />
+            </div>
+            <div class="text-div">
+              <h2 class="title">Multi-Post Stories</h2>
+              <ul class="tech-btns">
+                <li>CANOPY</li>
+                <li class="common">Back End Dev</li>
+                <li class="common">2015</li>
+              </ul>
+
+              <p class="paragraph">
+                A daily selection of privately personalized reads no accounts or
+                sign-ups required.
+              </p>
+              <ul class="languages">
+                <li>Html</li>
+                <li>Css</li>
+                <li>Javascript</li>
+              </ul>
+              <a class="see-project" data-index-number="2" href=""
+                >See Project</a
+              >
+            </div>
+          </li>
+        </ul>
+      </section>
+      <section class="About-Myself">
+        <div class="grid">
+          <div class="about">
+            <div class="head">
+              <h1 class="details">About Myself</h1>

```

Comment:
- [ ] Kindly change your h1 tag to h2, your design should have only one h1 tag 🤝